### PR TITLE
Improves `getAddress` types and adds isUInt4 util

### PR DESCRIPTION
**Pull request by douglance**

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -106,7 +107,7 @@ export class Client {
     /** The name of the client. */
     name?: string;
     /** The private key of the client.*/
-    privKey?: Buffer;
+    privKey?: Buffer | string;
```
Comment (alex-miller-0) [inline src/client.ts:110]: ??

Pretty sure this needs to be a buffer

**Review by alex-miller-0 — APPROVED** [review]
Looks good but thar be conflicts

**Comment by alex-miller-0** [comment]
@douglance please rebase this against dev rather than creating a merge commit in this branch

**Comment by douglance** [comment]
@alex-miller-0 [eac0cf3](https://github.com/GridPlus/gridplus-sdk/pull/368/commits/eac0cf3c22013cad418dab467700ae27d46de0a7)

**Review by alex-miller-0 — APPROVED** [review]

**Pull request merged**